src/pages/TeamSettings/FlowConcurrency.vue

```diff
@@ -282,323 +282,325 @@ export default {
-      <v-btn
-        color="primary"
-        class="white--text"
-        data-cy="add-flow-concurrency-limit"
-        large
-        @click="openAddDialog"
-      >
-        <v-icon left>
-          add
-        </v-icon>
-        Add Label
-      </v-btn>
-    </template>
-
-    <template v-if="!isEligible" #alerts>
-      <v-alert
-        class="mx-auto"
-        border="left"
-        colored-border
-        elevation="2"
-        type="warning"
-        tile
-        icon="lock"
-        max-width="600"
-      >
-        Your plan doesn't include flow concurrency limiting.
-        <ExternalLink href="/plans">Upgrade</ExternalLink> to get access to flow
-        concurrency and lots of other cool features!
-      </v-alert>
-    </template>
-
-    <template v-else-if="!permissionsCheck" #alerts>
-      <v-alert
-        class="mx-auto"
-        border="left"
-        colored-border
-        elevation="2"
-        type="warning"
-        tile
-        icon="lock"
-        max-width="600"
-      >
-        Only team administrators can manage flow concurrency limits.
-      </v-alert>
-    </template>
-
-    <v-text-field
-      v-if="!$vuetify.breakpoint.mdAndUp"
-      v-model="search"
-      class="rounded-0 elevation-1 mb-1"
-      solo
-      dense
-      hide-details
-      single-line
-      placeholder="Search by flow label or limit"
-      prepend-inner-icon="search"
-      autocomplete="new-password"
-    ></v-text-field>
-
-    <v-card v-if="isEligible" tile>
-      <v-card-text class="pa-0">
-        <div v-if="$vuetify.breakpoint.mdAndUp" class="py-1 mr-2 flex">
-          <v-text-field
-            v-model="search"
-            class="rounded-0 elevation-1"
-            solo
-            dense
-            hide-details
-            single-line
-            placeholder="Search by flow label or limit"
-            prepend-inner-icon="search"
-            autocomplete="new-password"
-            :style="{
-              'max-width': $vuetify.breakpoint.mdAndUp ? '360px' : null
-            }"
-          ></v-text-field>
-        </div>
-
-        <v-data-table
-          fixed-header
-          data-cy="label-table"
-          class="elevation-2 rounded-0 truncate-table"
-          :headers="headers"
-          :header-props="{ 'sort-icon': 'arrow_drop_up' }"
-          :items="labelsWithUsage"
-          :items-per-page="10"
-          :search="search"
-          :loading="loadingKey > 0"
-          :footer-props="{
-            showFirstLastPage: true,
-            itemsPerPageOptions: [10, 15, 20, -1],
-            firstIcon: 'first_page',
-            lastIcon: 'last_page',
-            prevIcon: 'keyboard_arrow_left',
-            nextIcon: 'keyboard_arrow_right'
-          }"
-          no-results-text="No concurrency limits found. Try expanding your search?"
-          no-data-text="This team has not set any flow concurrency limits yet."
+  <div v-intersect="{ handler: onIntersect }">
```

Comment:
Changes starts here. I removed `v-intersect` from `MangementLayout` and added it to the div. Also, it seems my formatting got messed up so not sure how to fix that?